docs/products/kafka/howto/configure-log-cleaner.rst

```diff
@@ -0,0 +1,34 @@
+Configure log cleaner for topic compaction
+==========================================
+
+The log cleaner is the process ensuring only the most recent value for a certain message key is kept within a partition for :doc:`compacted topics <../concepts/log-compaction>`.  In Aiven for Apache Kafka the log cleaner is enabled, but the log compaction is disabled by default. To enable log compaction, follow these steps:
+
+#. Log into the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+#. Scroll down to *Advanced configuration* and click on **Add configuration option**.
+#. Find ``log.cleanup.policy`` in the list and select it.
+#. Set the value to ``compact``.
+
+.. warning:: This change will affect all topics in the cluster that do not have a configuration override in place.
+
+Log cleaning frequency and delay
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before the cleaning begins, the cleaner thread will inspect the logs to find those with highest **dirty ratio** calculated as the number of bytes in the head vs the total number of bytes in the log (tail + head); you can read more about head and tail definition in the :doc:`compacted topic documentation <../concepts/log-compaction>`. The ratio provides an estimate of how many duplicated keys are present in a topic, and therefore need to be compacted.
+
+.. Tip::
+
+    For the log cleaner to start compacting a topic, the dirty ratio needs to be bigger than a threshold set to 50% by default. You can change this value by modifying the property ``kafka.log_cleaner_min_cleanable_ratio`` in the *Advanced configuration* section of the service overview.
+
+The log cleaner can be configured to leave some amount of not compacted "head" of the log by setting compaction time lag. You can achieve this by setting two additional properties from the  *Advanced configuration*:
+
+-  ``log.cleaner.min.compaction.lag.ms`` : setting to a value greater than 0 will prevent log cleaner from compacting messages with an age newer than a minimum message age. Allows to delay compacting records.
```

Comment:
does it make sense to connect those two sentences with an **and**? What does allows to delay compacting records: the process itself or the setting the value greater than 0?